src/content/docs/en/reference/modules/astro-assets.mdx

```diff
@@ -47,19 +48,224 @@ import myImage from "../assets/my_image.png"; // Image is 1600x900
-- alt (required)
-- width and height (required for `public/` and remote images)
-- format
-- quality
-- densities
-- widths
+#### Image properties
 
-In addition to the properties above, the `<Image />` component accepts all properties accepted by the HTML `<img>` tag.
+The `<Image />` component accepts all properties accepted by the HTML `<img>` tag in addition to the properties described below.
 
-See more in the [Images Guide](/en/guides/images/#image--astroassets).
+##### src (required)
+
+<p>
+
+**Type:** `ImageMetadata | string | Promise<{ default: ImageMetadata }>`
+</p>
+
+The format of the `src` value of your image file depends on where your image file is located:
+
+- **Local images in `src/`** - you must **also import the image** using a relative file path or configure and use an [import alias](/en/guides/imports/#aliases). Then use the import name as the `src` value:
+
+  ```astro title="src/pages/index.astro" "myImportedImage" "{myImportedImage}"
+  ---
+  import { Image } from 'astro:assets';
+  import myImportedImage from '../assets/my-local-image.png';
+  ---
+  <Image src={myImportedImage} alt="descriptive text" />
+  ```
+
+- **Images in the `public/` folder** -  use the image's **file path relative to the public folder**:
+
+  ```astro title="src/pages/index.astro" '"/images/my-public-image.png"'
+  ---
+  import { Image } from 'astro:assets';
+  ---
+  <Image
+    src="/images/my-public-image.png"
+    alt="descriptive text"
+    width="200"
+    height="150"
+  />
+  ```
+
+- **Remote images** - use the image's **full URL** as the property value:
+
+  ```astro title="src/pages/index.astro" '"https://example.com/remote-image.jpg"'
+  ---
+  import { Image } from 'astro:assets';
+  ---
+  <Image
+    src="https://example.com/remote-image.jpg"
+    alt="descriptive text"
+    width="200"
+    height="150"
+  />
+  ```
+
+##### alt (required)
+
+<p>
+
+**Type:** `string`
+</p>
+
+Use the required `alt` attribute to provide a string of [descriptive alt text](https://www.w3.org/WAI/tutorials/images/) for images.
+
+If an image is merely decorative (i.e. doesn't contribute to the understanding of the page), set `alt=""` so that screen readers and other assistive technologies know to ignore the image.
+
+##### width and height (required for images in `public/`)
+
+<p>
+
+**Type:** `number | undefined`
+</p>
+
+These properties define the dimensions to use for the image.
+
+When using images in their original aspect ratio, `width` and `height` are optional. These dimensions can be automatically inferred from image files located in `src/`. For remote images, add [the `inferSize` attribute set to `true`](#infersize) on the `<Image />` or `<Picture />` component or use [`inferRemoteSize()` function](#inferremotesize).
+
+However, both of these properties are required for images stored in your `public/` folder as Astro is unable to analyze these files.
+
+##### densities
+
+<p>
+
+**Type:** `(number | `${number}x`)[] | undefined`<br />
```

Comment:
I think this is the type that is causing problems with `Check links`. Maybe we could use the `code` tag instead:
```suggestion
**Type:** <code>(number | `${number}x`)[] | undefined<code><br />
```